job_spec/INSAR_ISCE_BURST.yml

```diff
@@ -1,23 +1,38 @@
 INSAR_ISCE_BURST:
   required_parameters:
-    - granules
+    - reference
+    - secondary
   parameters:
-    granules:
+    reference:
       default:  '""'
       api_schema:
         type: array
-        minItems: 2
-        maxItems: 2
+        minItems: 1
+        maxItems: 1
         example:
           - S1_136231_IW2_20200604T022312_VV_7C85-BURST
```

Comment:
I wonder if we need an example for the array since we also have an example for the item. Not sure what it would look like in the Swagger UI if we removed the array example.

Reply:
We include both for all other jobs. I'm not sure if it does anything or not.

Reply:
Sounds good, let's leave it then.